x-pack/plugins/security_solution/cypress/tasks/rules_bulk_edit.ts

```diff
@@ -166,3 +177,51 @@ export const checkTagsInTagsFilter = (tags: string[]) => {
       cy.wrap($el).should('have.text', tags[index]);
     });
 };
+
+export const typeScheduleInterval = (interval: string) => {
+  cy.get(UPDATE_SCHEDULE_INTERVAL_INPUT)
+    .find('input')
+    .type('{selectAll}')
+    .type(interval.toString())
+    .blur();
+};
+export const typeScheduleLookback = (lookback: string) => {
+  cy.get(UPDATE_SCHEDULE_LOOKBACK_INPUT)
+    .find('input')
+    .type('{selectAll}', { waitForAnimations: true })
+    .type(lookback.toString(), { waitForAnimations: true })
+    .blur();
+};
+
+type TimeUnit = 'Seconds' | 'Minutes' | 'Hours';
+export const setScheduleIntervalTimeUnit = (timeUnit: TimeUnit) => {
+  cy.get(UPDATE_SCHEDULE_INTERVAL_INPUT).within(() => {
+    cy.get(UPDATE_SCHEDULE_TIME_UNIT_SELECT).select(timeUnit);
+  });
+};
+
+export const setScheduleLookbackTimeUnit = (timeUnit: TimeUnit) => {
+  cy.get(UPDATE_SCHEDULE_LOOKBACK_INPUT).within(() => {
+    cy.get(UPDATE_SCHEDULE_TIME_UNIT_SELECT).select(timeUnit);
+  });
+};
+
+export const assertUpdateScheduleWarningExists = (expectedNumberOfNotMLRules: number) => {
+  cy.get(RULES_BULK_EDIT_SCHEDULES_WARNING).should(
+    'have.text',
+    `You're about to apply changes to ${expectedNumberOfNotMLRules} selected rules. The changes you made will be overwritten to the existing Rule schedules and additional look-back time (if any).`
```

Comment:
I think this is just test UI copy, but to match actual UI copy, edit the second sentence to "The changes you make will overwrite the existing rule schedules and additional look-back time (if any)."